fdbclient/ServerKnobs.cpp

```diff
@@ -289,6 +289,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( DD_STORAGE_WIGGLE_PAUSE_THRESHOLD,                      10 ); if( randomize && BUGGIFY ) DD_STORAGE_WIGGLE_PAUSE_THRESHOLD = 1000;
 	init( DD_STORAGE_WIGGLE_STUCK_THRESHOLD,                      20 );
 	init( DD_BUILD_EXTRA_TEAMS_OVERRIDE,                          10 ); if( randomize && BUGGIFY ) DD_BUILD_EXTRA_TEAMS_OVERRIDE = 2;
+	init( ENABLE_STORAGE_QUEUE_AWARE_TEAM_SELECTION,           false );
```

Comment:
Maybe add a buggify to true (in a follow up PR).

Reply:
Yes. Thank you for the suggestion!